SSA/Projects/InstCombine/ComWrappers.lean

```diff
@@ -0,0 +1,244 @@
+import SSA.Projects.InstCombine.Base
+
+/- Wrapper around Com, Expr constructors to easily hand-write IR -/
+namespace ComWrappers
+
+local macro "offset_in_bounds" : tactic =>
+  `(tactic|
+      (
+      simp [Ctxt.snoc]
+      )
+   )
```

Comment:
Isn't this the same as the auto-tactic used in `HVector.getN`, why not re-use that? Or, add `Ctxt.snoc` to the simp-set used by `get_elem_tactic` and just use that (since morally we're doing the same kind of index-in-bounds checking)

Reply:
Done.